src/frontend/components/Kontoinformasjon/Kontoinformasjon.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { Alert, Heading, Label, Loader, VStack } from '@navikt/ds-react';
+import { RessursStatus } from '@navikt/familie-typer';
+
+import { useApp } from '../../context/AppContext';
+import { ESanitySteg, Typografi } from '../../typer/sanity/sanity';
+import TekstBlock from '../Felleskomponenter/Sanity/TekstBlock';
+
+const Kontoinformasjon: React.FC = () => {
+    const { kontoinformasjon, tekster, plainTekst } = useApp();
+
+    const kvitteringstekster = tekster()[ESanitySteg.KVITTERING];
+
+    return (
+        <VStack gap="6">
+            {kontoinformasjon.status === RessursStatus.SUKSESS && (
+                <>
+                    <Heading level="3" size="xsmall">
+                        {plainTekst(kvitteringstekster.kontonummerTittel)}
+                    </Heading>
+                    <Label as="p">{kontoinformasjon.data.kontonummer}</Label>
+                    <TekstBlock
+                        typografi={Typografi.BodyLong}
+                        block={kvitteringstekster.redigerKontonummerLenke}
+                    />
+                </>
+            )}
+
+            {kontoinformasjon.status === RessursStatus.HENTER && (
+                <>
+                    <Heading level="3" size="xsmall">
+                        {plainTekst(kvitteringstekster.kontonummerTittel)}
+                    </Heading>
+                    <Loader title={plainTekst(kvitteringstekster.henterKontonummer)} />
+                </>
+            )}
```

Comment:
I og med at henting av kontonummer skjer med en gang man er logget inn og man ikke kan komme seg til kvitteringssiden uten å være logget inn, skal det nok en del til for at vi havner i tilstanden hvor kontonummer hentes mens man står på kvitteringssiden? 🤔 Det gjør absolutt ikke noe at det er slik da så trenger ikke endre noe 😊 Bare en observasjon!